pyproject.toml

```diff
@@ -43,7 +43,7 @@ dependencies = [
     "python-json-logger>=2.0.0",
     "pytimeparse>=1.1.8,<2.0.0",
     "pyyaml!=6.0.0,!=5.4.0,!=5.4.1", # pyyaml is broken with cython 3: https://github.com/yaml/pyyaml/issues/601
-    "requests>=2.18.4,<3.0.0,!=2.32.0,!=2.32.1,!=2.32.2",
+    "requests>=2.18.4,<3.0.0,!=2.32.0,!=2.32.1,!=2.32.2,!=2.32.3",
```

Comment:
Is there any issue in 2.32.3?

Reply:
error: https://github.com/flyteorg/flytekit/actions/runs/9297630095/job/25588079504